linux.md

```diff
@@ -198,10 +198,9 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
     sudo apt update
     sudo apt install postgresql-16
-    echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
+    echo -e "export PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
     PGDATA=$(dirname "$(sudo -u postgres psql --tuples-only --pset format=unaligned --command "SHOW config_file;")")
     source ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
-    perl -i -pe 's/^[#\s]*(lc_messages|lc_time)\s*=.+$/\1 = '\''en_US.UTF-8'\''/' "$PGDATA/postgresql.conf"
```

Comment:
Setting `lc_messages` and `lc_time` in the `postgresql.conf` file is not needed when the `LANG` is set as the Postgres server will use the value of this whenever it's set and only default back to the values in the `postgresql.conf` file when `LANG` is not set

Source
[Postgres docs](https://www.postgresql.org/docs/current/locale.html#:~:text=The%20values%20that%20are%20chosen%20by%20initdb%20are%20actually%20only%20written%20into%20the%20configuration%20file%20postgresql.conf%20to%20serve%20as%20defaults%20when%20the%20server%20is%20started.%20If%20you%20remove%20these%20assignments%20from%20postgresql.conf%20then%20the%20server%20will%20inherit%20the%20settings%20from%20its%20execution%20environment.)
[Postgres docs](https://www.postgresql.org/docs/current/locale.html#:~:text=LC_ALL%2C%20LC_COLLATE%20(or%20the%20variable%20corresponding%20to%20the%20respective%20category)%2C%20LANG)